tools/octoclock_test/octoclock_test.cpp

```diff
@@ -17,6 +17,7 @@
 #include <uhd/utils/safe_main.hpp>
```

Comment:
Is this tool valuable at this stage of borealis? If so, perhaps rewrite to python? I find that since python doesn't rely on a build system then these tools need less maintenance.

Reply:
nah I just used it for checking timing on queries to the octoclock, you can ignore this I think.